src/test/java/org/opensearch/neuralsearch/query/HybridQueryIT.java

```diff
@@ -395,7 +446,200 @@ public void testIndexWithNestedFields_whenHybridQueryIncludesNested_thenSuccess(
             assertNotNull(total.get("relation"));
             assertEquals(RELATION_EQUAL_TO, total.get("relation"));
         } finally {
-            wipeOfTestResources(TEST_MULTI_DOC_INDEX_WITH_NESTED_TYPE_NAME_ONE_SHARD, null, modelId, SEARCH_PIPELINE);
+            wipeOfTestResources(TEST_MULTI_DOC_INDEX_WITH_NESTED_TYPE_NAME_ONE_SHARD, null, null, SEARCH_PIPELINE);
+        }
+    }
+
+    @SneakyThrows
+    public void testPipelineAggs_whenConcurrentSearchEnabled_thenSuccessful() {
```

Comment:
I would probably move all these ITs related to aggregation in a separate IT class. This will ensure that we are not bloating this class with more and more tests, plus we have a separate class for Aggregations use case.

Reply:
ack, let me refactor into separate class. My idea was to add more test for aggs only, and put them into a separate package so that's easier to exclude them from all test suite